rest/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>liquor-bottles-registry-parent</artifactId>
+        <groupId>cz.fi.muni.pa165</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>liquor-bottles-registry-rest</artifactId>
+    <packaging>war</packaging>
+    <name>Rest layer</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.tomcat.maven</groupId>
+                <artifactId>tomcat7-maven-plugin</artifactId>
+                <version>2.2</version>
+                <configuration>
+                    <path>/pa165/rest</path>
```

Comment:
Myslim, ze my potrebujeme aby v jednom module bol aj rest aj angular, to znamena, ze to musime namapovat na /pa165 a rest namapovat na /pa165/rest. To nastavis v Initializer#getServletMappings. Alebo som to pochopil zle?